p2p/pkg/preconfirmation/tracker/tracker_test.go

```diff
@@ -132,7 +132,7 @@ func TestTracker(t *testing.T) {
 			continue
 		}
 
-		err = publishEncCommitment(evtMgr, &pcABI, preconf.PreconfcommitmentstoreEncryptedCommitmentStored{
+		err = publishEncCommitment(evtMgr, &pcABI, preconf.PreconfcommitmentstoreUnopenedCommitmentStored{
```

Comment:
Does changing the publish func naming make sense here? 
```suggestion
		err = publishUnopenedCommitment(evtMgr, &pcABI, preconf.PreconfcommitmentstoreUnopenedCommitmentStored{
```